CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ These changes are available on the `master` branch, but have not yet been releas
 
 ### Added
 
+- Added `member` to the `raw_reaction_remove` event.
+  ([#2412](https://github.com/Pycord-Development/pycord/pull/2412))
```

Comment:
We add changelog entries at the bottom of sections instead of the top usually, I'll fix this one for you